media/js/base/protocol/init-navigation.js

```diff
@@ -41,6 +41,62 @@
         }
     }
 
+    var nav = document.querySelector('.mzp-c-navigation');
```

Comment:
Nit: suggest putting all this code into a function that gets called after the navigation initializes? The more stuff we put in front of that code, the longer it takes to run for everyone. Seeing as this is applicable only to Firefox users, it's probably safe for it to come after?